hs_restclient/util.py

```diff
@@ -0,0 +1,10 @@
+
+def is_sequence(obj):
+    """ Determine whether an object is a sequence, but not a string.
+
+    :param obj: Object to test
+    :return: True if the object is a sequence type, but not a string.
+    """
+    return (not hasattr(obj, "strip") and
+            hasattr(obj, "__getitem__") or
+            hasattr(obj, "__iter__"))
```

Comment:
This whole module for this one function which you use in exactly one line. It would probably be easier to just trust your users to follow the documentation. If they pass invalid resource types, the request is going to fail anyway. I think type checking in Python always requires a **really** good reason.